ui/src/app/components/session-listing/session-listing.component.html

```diff
@@ -11,7 +11,8 @@ <h3>Session history</h3>
           <div class="table-header-container">
             <span>Session Name</span>
             <button mat-icon-button (click)="toggleFilterDisplay('sessionName')">
-              <mat-icon>filter_list</mat-icon>
+              <mat-icon *ngIf="displayFilter['sessionName']">keyboard_arrow_up</mat-icon>
```

Comment:
In case no rows match the filter the alignment of header changes because Resume & Download buttons are not visible. We need to fix the css